### DISCUSSION TITLE
vue的子应用build过程中出现Syntax Error: TypeError: Cannot read property 'prototype' of undefined

### DISCUSSION THREAD

**Discussion by ywfgg** [jd-opensource/micro-app#843] [Q&A]
![1688460370078](https://github.com/micro-zoe/micro-app/assets/23191124/8925bd94-460c-4079-b50a-72182557966c)

**Comment by raoenhui** [comment]
ts语法问题需要自己修复哈，可以看看prototype为什么没定义
